### PR TITLE
Enable tests

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

File: shared/src/main/scala/minitest/api/SourceLocation.scala [hunk 1]
```diff
@@ -34,7 +34,7 @@ object SourceLocation {
     val pos = tasty.rootPosition
     val fileName = pos.sourceFile.getFileName.toString
     val path = pos.sourceFile.getParent.toString
-    val line = pos.startLine
+    val line = pos.startLine + 1
```
Comment (allanrenucci) [inline shared/src/main/scala/minitest/api/SourceLocation.scala]: Why is `+ 1` needed? is `pos.startLine` broken?

File: build.sbt [hunk 2]
```diff
@@ -181,6 +181,6 @@ lazy val dottySettings = List(
 )
 
 TaskKey[Unit]("dottyCompile") := {
-  compile.in(minitestJVM, Test).value
-  compile.in(lawsJVM, Compile).value // Test fail to compile
```
Comment (allanrenucci) [inline build.sbt]: Why is the comment removed? Are the test in `lawsJVM` compiling? If so, the line should be changed to:
```scala
compile.in(lawsJVM, Test).value
// or if the tests run as well:
test.in(lawsJVM, Test).value
```

File: shared/src/main/scala/minitest/api/SourceLocation.scala (hunk 1 above)
Comment (nicolasstucki) [reply shared/src/main/scala/minitest/api/SourceLocation.scala]: No, `startLine` is the index of the start line but the IDEs start with line 1.

File: build.sbt (hunk 2 above)
Comment (nicolasstucki) [reply build.sbt]: That currently fails with 
```
[error] undefined: new <notype> # 9166: TermRef(TypeRef(TermRef(ThisType(TypeRef(NoPrefix,module class reflect)),module annotation),EnableReflectiveInstantiation),<init>) at frontend
```
an unrelated issue.

Comment (nicolasstucki) [reply build.sbt]: That is a scala.js annotation.

Comment (nicolasstucki) [reply build.sbt]: Added comment again

**Pull request merged**